.pre-commit-config.yaml

```diff
@@ -0,0 +1,64 @@
+# https://pre-commit.com/
+repos:
+  # General hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+        args: ["--markdown-linebreak-ext=md"]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
+        name: doc format
+        files: ^doc/
```

Comment:
This would exclude e.g. backend/README.md, won't it?

Reply:
True.
It would omit the root README.md as well.
I will adjust it.

Reply:
see
https://github.com/ElektraInitiative/PermaplanT/pull/160/commits/96b260a9ffc2290f8fda6009241f118b3a5f19d2